designs/FloatingVersions-In-PMUI.md

```diff
@@ -0,0 +1,104 @@
+
+# Floating Versions in the Package Manager UI
+
+* Status: In Review
+* Author(s): [Martin Ruiz](https://github.com/martinrrm)
+* Issue: [3788](https://github.com/NuGet/Home/issues/3788) PackageReference with a star in version is displayed under the updates tab/incorrectly represented in the installed tab
+
+## Problem Background
+
+Currently the user can specify a version range or a floating version, but the current behavior of the package manager UI has poor support for this, displaying an incorrect installed version. If the user indicates a floating version in any package (eg `5.*`) the PM UI will display the minimum version possible (`5.0.0`) as installed even though the resolved version could be different of the resolved version in the command `dotnet list package`. Furthermore the user cannot specify a version range free hand.
+
+### Current behavior
+
+### Updates Tab
+
+With the current behavior even if the user indicated a floating version, such as `*`, for the package version, the UI will tell the user that there is an update available which is misleading.
+
+#### Project File and Solution Explorer
+
+![SolutionExplorer](../resources/FloatingVersionsInPMUI/ProjectFile&SolutionExplorer.png)
+
+#### Project Manager UI
+
+![Package UI](../resources/FloatingVersionsInPMUI/PackageManagerUI.png)
+
+![Project UI](../resources/FloatingVersionsInPMUI/PackageManagerForSolution.png)
+
+```
+The requested and installed versions are not correct because dotnet list package shows a different installed version.
```

Comment:
We should remain as true as we can to the `dotnet` cli as possible. `Requested` and `Resolved` would be better words than what we have today (Installed / Version).